README.md

```diff
@@ -337,7 +350,7 @@ $ node-debug --web-host 127.0.0.2 app
 ``` 
 Pass `--option=value` to debugging process:
 ```
-$ node-debug app --option value
+$ node-debug --option value app
```

Comment:
This looks suspicious to me. IIRC the arguments *before* the script file are processed by Node Inspector and/or node-debug. Only the arguments *after* the script file are passed to the debugged node process. @3y3: am I wrong?